stdlib/http/src/main/ballerina/src/http/Module.md

```diff
@@ -1,5 +1,3 @@
-## Module overview
-
 This module provides an implementation for connecting and interacting with HTTP, HTTP2, and WebSocket endpoints. The module facilitates two types of endpoints as ‘Client’ and ‘Listener’.
 
 ### Client endpoints
```

Comment:
```suggestion
### Client
```

Reply:
Shall we remove the remaining occurrences of endpoint terminology as it has been removed sometimes back?

Reply:
+1